srcpkgs/libnma/template

```diff
@@ -40,5 +40,8 @@ libnma-devel_package() {
 		vmove usr/share/gir-1.0
 		vmove usr/share/vala
 		vmove usr/share/gtk-doc
+
+		# This file is now provided by network-manager-applet
+		rm -f ${DESTDIR}/usr/share/glib-2.0/schemas/org.gnome.nm-applet.gschema.xml
```

Comment:
Move this to `post_install()` and remove `-f` because install can run only once and we want it to fail if the file doesn't exist.